### PR TITLE
Add stateless tests for `SHOW FUNCTIONS`

### PR DESCRIPTION
**Pull request by lianghanzhen**
I hereby agree to the terms of the CLA available at: https://databend.rs/policies/cla/

## Summary

Add stateless tests for `SHOW FUNCTIONS`

## Changelog

- Build/Testing/CI

## Test Plan

Stateless Tests

### CODE REVIEW

File: tests/suites/0_stateless/README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ xx_yyyy_[test_name]
 * 10 -- for Describe tests, [example](10_0000_describe_table.sql)
 * 11 -- for Show tables, [example](11_0000_show_tables.sql)
 * 12 -- for insert into select, [example](12_0000_insert_into_select.sql)
+* 13 -- for Show functions, [example](14_0000_show_functions.sql)
```
Comment (BohuTANG) [inline tests/suites/0_stateless/README.md:25]: We can merge all show to 11 `11 -- for Show commands`

Comment (lianghanzhen) [reply tests/suites/0_stateless/README.md]: Updated

**Comment by BohuTANG** [comment]
/lgtm

Thank you!

**Comment by lianghanzhen** [comment]
@BohuTANG Can you help rerun the unit tests? I think it's false positive.

**Comment by BohuTANG** [comment]
Thank you @lianghanzhen !

**Pull request merged**